featurebyte/session/base.py

```diff
@@ -48,7 +48,7 @@
 from featurebyte.query_graph.sql.source_info import SourceInfo
 
 INTERACTIVE_SESSION_TIMEOUT_SECONDS = 30
-NON_INTERACTIVE_SESSION_TIMEOUT_SECONDS = 120
+NON_INTERACTIVE_SESSION_TIMEOUT_SECONDS = 600
```

Comment:
Should we consider setting an even longer timeout to cater for longer startup time?